src/Files.Uwp/Views/Pages/PropertiesSecurityAdvanced.xaml

```diff
@@ -35,14 +35,6 @@
                 x:Key="BoolToWidthConverter"
                 FalseValue="0"
                 TrueValue="*" />
-
-            <AcrylicBrush
```

Comment:
What's going on here?

Reply:
Acrylic brush defined but not used anywhere, so removed.